src/stores/scheduling.ts

```diff
@@ -156,6 +162,10 @@ export const allowedSchedulingGoalSpecs: Readable<SchedulingGoalPlanSpecificatio
     $schedulingGoalSpecifications.filter(({ goal_metadata: goalMetadata }) => goalMetadata !== null),
 );
 
+export const allowedSchedulingGoalInvocations = derived([schedulingGoalInvocations], ([$schedulingGoalInvocations]) =>
```

Comment:
I think this might be redundant. This doesn't seem different from `allowedSchedulingGoalSpecs` right?